### PR TITLE
Update rubygem-graphql spec

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/foreman/rubygem-graphql/rubygem-graphql.spec [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 Version: 1.8.14
-Release: 2%{?dist}
+Release: 3%{?dist}
 Summary: A GraphQL language and runtime for Ruby
 Group: Development/Languages
 License: MIT
 URL: http://github.com/rmosolgo/graphql-ruby
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+Autoreq: 0
```
[the remark this answers is not in the document]
Comment (evgeni) [reply packages/foreman/rubygem-graphql/rubygem-graphql.spec:17]: then please drop it there ;)

[the remark this answers is not in the document]
Comment (ekohl) [reply packages/foreman/rubygem-graphql/rubygem-graphql.spec:17]: https://github.com/theforeman/foreman-packaging/pull/6412

**Comment by evgeni** [comment]
```
ERROR:  While executing gem ... (RDoc::Error)
    error generating GraphQL/Schema/Member/BuildType.html: no implicit conversion of nil into String (TypeError)
```

**Comment by ehelms** [comment]
> ```
> ERROR:  While executing gem ... (RDoc::Error)
>     error generating GraphQL/Schema/Member/BuildType.html: no implicit conversion of nil into String (TypeError)
> ```

Turns out we were working around this, and hence the difference in this spec file versus the template. In my Ruby 2.7 testing, this issue goes away. So I will keep this here for now but will fold it into my Ruby 2.7 PRs and close this once I do.

**Comment by ehelms** [comment]
This has been incoporated within the Ruby 2.7 updates

**Pull request closed without merging**